### PR TITLE
add new theme "griddy"

### PR DESCRIPTION
**Pull request by sira313**
- [x] Read the [theme publishing doc](https://hexo.io/docs/themes#Publishing) or [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).

### CODE REVIEW

File: source/_data/themes.yml [hunk 1]
```diff
@@ -1,3 +1,16 @@
+- name: Griddy
+  description: Hexo theme for artist & photographer showing their work
+  link: https://github.com/sira313/hexo-theme-griddy
+  preview: https://aflasio.netlify.com
+  tags:
+    - simple
+    - visual
+    - three_column
```
[the remark this answers is not in the document]
Comment (sira313) [reply source/_data/themes.yml]: 4 column for large desktop. Should i edit the tag?

[the remark this answers is not in the document]
Comment (YoshinoriN) [reply source/_data/themes.yml]: Either is fine :)
I will merge this if you ok.

[the remark this answers is not in the document]
Comment (sira313) [reply source/_data/themes.yml]: it's fine if you said so, thank you

**Comment by YoshinoriN** [comment]
@sira313 
Merged! Thank you for your contribution :)

**Pull request merged**